<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="11" failed="1" ignored="0" total="20" passed="8">
  <reporter-output>
  </reporter-output>
  <suite name="iBridge" duration-ms="1160260" started-at="2022-07-18T12:14:22Z" finished-at="2022-07-18T12:33:42Z">
    <groups>
    </groups>
    <test name="iBridge18" duration-ms="1160260" started-at="2022-07-18T12:14:22Z" finished-at="2022-07-18T12:33:42Z">
      <class name="com.iBridge.testcases.TC_CatSignOffApplicable_01">
        <test-method status="PASS" signature="setup(java.lang.String)[pri:0, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="setup" is-config="true" duration-ms="7594" started-at="2022-07-18T12:14:22Z" finished-at="2022-07-18T12:14:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="verifyiBridgeCatLoginDetails()[pri:1, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="verifyiBridgeCatLoginDetails" duration-ms="5592" started-at="2022-07-18T12:14:30Z" finished-at="2022-07-18T12:14:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyiBridgeCatLoginDetails -->
        <test-method status="PASS" signature="verifyBritishClientSignOffApplicableYes()[pri:2, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="verifyBritishClientSignOffApplicableYes" duration-ms="9476" started-at="2022-07-18T12:14:36Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.verifyiBridgeCatLoginDetails" finished-at="2022-07-18T12:14:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyBritishClientSignOffApplicableYes -->
        <test-method status="PASS" signature="verifyInitiateCaseIndividuallyExtended()[pri:3, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="verifyInitiateCaseIndividuallyExtended" duration-ms="276340" started-at="2022-07-18T12:14:45Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.verifyBritishClientSignOffApplicableYes" finished-at="2022-07-18T12:19:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyInitiateCaseIndividuallyExtended -->
        <test-method status="PASS" signature="verifyPendingSignOffBucket()[pri:4, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="verifyPendingSignOffBucket" duration-ms="16082" started-at="2022-07-18T12:19:21Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.verifyInitiateCaseIndividuallyExtended" finished-at="2022-07-18T12:19:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyPendingSignOffBucket -->
        <test-method status="PASS" signature="verifyBritishClientSignOffApplicableNo()[pri:5, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="verifyBritishClientSignOffApplicableNo" duration-ms="14602" started-at="2022-07-18T12:19:38Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.verifyPendingSignOffBucket" finished-at="2022-07-18T12:19:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyBritishClientSignOffApplicableNo -->
        <test-method status="PASS" signature="submitCaseIndividuallyExtended()[pri:6, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="submitCaseIndividuallyExtended" duration-ms="247267" started-at="2022-07-18T12:19:52Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.verifyBritishClientSignOffApplicableNo" finished-at="2022-07-18T12:23:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitCaseIndividuallyExtended -->
        <test-method status="PASS" signature="verifyBritishClientRouteToCandidateApplicableYes()[pri:7, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="verifyBritishClientRouteToCandidateApplicableYes" duration-ms="13666" started-at="2022-07-18T12:23:59Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.submitCaseIndividuallyExtended" finished-at="2022-07-18T12:24:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyBritishClientRouteToCandidateApplicableYes -->
        <test-method status="PASS" signature="candidateInitiateCaseIndividuallyExtended()[pri:8, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="candidateInitiateCaseIndividuallyExtended" duration-ms="460625" started-at="2022-07-18T12:24:13Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.verifyBritishClientRouteToCandidateApplicableYes" finished-at="2022-07-18T12:31:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- candidateInitiateCaseIndividuallyExtended -->
        <test-method status="FAIL" signature="routeToCandidateRaiseInsuffAndFulfil()[pri:9, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="routeToCandidateRaiseInsuffAndFulfil" duration-ms="108761" started-at="2022-07-18T12:31:54Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.candidateInitiateCaseIndividuallyExtended" finished-at="2022-07-18T12:33:42Z">
          <exception class="org.openqa.selenium.ElementClickInterceptedException">
            <message>
              <![CDATA[element click intercepted: Element <select name="Checks[9598643][INSUFF_FULFILL_ACTION]" id="CaseChecks">...</select> is not clickable at point (467, 10). Other element would receive the click: <a href="javascript:void(0);">...</a>
  (Session info: chrome=101.0.4951.54)
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:19:58.91Z'
System info: host: 'ARS-NBT-LAP-068', ip: '192.168.79.30', os.name: 'Linux', os.arch: 'amd64', os.version: '5.13.0-51-generic', java.version: '1.8.0_312'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 101.0.4951.54, chrome: {chromedriverVersion: 101.0.4951.41 (93c720db8323..., userDataDir: /tmp/.com.google.Chrome.c1xdzQ}, goog:chromeOptions: {debuggerAddress: localhost:33303}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: LINUX, platformName: LINUX, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 631ae516c0d4a832bc55ac3c2bb7989b]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element <select name="Checks[9598643][INSUFF_FULFILL_ACTION]" id="CaseChecks">...</select> is not clickable at point (467, 10). Other element would receive the click: <a href="javascript:void(0);">...</a>
  (Session info: chrome=101.0.4951.54)
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:19:58.91Z'
System info: host: 'ARS-NBT-LAP-068', ip: '192.168.79.30', os.name: 'Linux', os.arch: 'amd64', os.version: '5.13.0-51-generic', java.version: '1.8.0_312'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 101.0.4951.54, chrome: {chromedriverVersion: 101.0.4951.41 (93c720db8323..., userDataDir: /tmp/.com.google.Chrome.c1xdzQ}, goog:chromeOptions: {debuggerAddress: localhost:33303}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: LINUX, platformName: LINUX, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 631ae516c0d4a832bc55ac3c2bb7989b
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:276)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:83)
at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at com.sun.proxy.$Proxy8.click(Unknown Source)
at com.iBridge.pageobject.InsufficientBridge.superAdminInsufficiencyRejectFulfillment(InsufficientBridge.java:1857)
at com.iBridge.testcases.TC_InsufficientBridge_01.verifyInsufficiencyRejectedFulfillment(TC_InsufficientBridge_01.java:260)
at com.iBridge.testcases.TC_CatSignOffApplicable_01.routeToCandidateRaiseInsuffAndFulfil(TC_CatSignOffApplicable_01.java:257)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.ElementClickInterceptedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- routeToCandidateRaiseInsuffAndFulfil -->
        <test-method status="SKIP" signature="verifyrouteToCandidateApplicableNo()[pri:10, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="verifyrouteToCandidateApplicableNo" duration-ms="0" started-at="2022-07-18T12:33:42Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.routeToCandidateRaiseInsuffAndFulfil" finished-at="2022-07-18T12:33:42Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_CatSignOffApplicable_01.verifyrouteToCandidateApplicableNo()[pri:10, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_CatSignOffApplicable_01.verifyrouteToCandidateApplicableNo()[pri:10, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyrouteToCandidateApplicableNo -->
        <test-method status="SKIP" signature="verifyrouteToCandidateButtonPresentOrNot()[pri:11, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="verifyrouteToCandidateButtonPresentOrNot" duration-ms="0" started-at="2022-07-18T12:33:42Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.verifyrouteToCandidateApplicableNo" finished-at="2022-07-18T12:33:42Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_CatSignOffApplicable_01.verifyrouteToCandidateButtonPresentOrNot()[pri:11, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_CatSignOffApplicable_01.verifyrouteToCandidateButtonPresentOrNot()[pri:11, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyrouteToCandidateButtonPresentOrNot -->
        <test-method status="SKIP" signature="verifyBritishClientNidPassport()[pri:12, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="verifyBritishClientNidPassport" duration-ms="0" started-at="2022-07-18T12:33:42Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.verifyrouteToCandidateButtonPresentOrNot" finished-at="2022-07-18T12:33:42Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_CatSignOffApplicable_01.verifyBritishClientNidPassport()[pri:12, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_CatSignOffApplicable_01.verifyBritishClientNidPassport()[pri:12, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyBritishClientNidPassport -->
        <test-method status="SKIP" signature="verifyBritishClientNidPassportAndPanCard()[pri:14, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="verifyBritishClientNidPassportAndPanCard" duration-ms="0" started-at="2022-07-18T12:33:42Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.verifyNidPassportDropDown" finished-at="2022-07-18T12:33:42Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_CatSignOffApplicable_01.verifyBritishClientNidPassportAndPanCard()[pri:14, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_CatSignOffApplicable_01.verifyBritishClientNidPassportAndPanCard()[pri:14, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyBritishClientNidPassportAndPanCard -->
        <test-method status="SKIP" signature="verifyNidPassportDropDown()[pri:13, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="verifyNidPassportDropDown" duration-ms="0" started-at="2022-07-18T12:33:42Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.verifyBritishClientNidPassport" finished-at="2022-07-18T12:33:42Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_CatSignOffApplicable_01.verifyNidPassportDropDown()[pri:13, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_CatSignOffApplicable_01.verifyNidPassportDropDown()[pri:13, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyNidPassportDropDown -->
        <test-method status="SKIP" signature="verifyNidVoterIdAndDrivingLicenseDropDown()[pri:15, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="verifyNidVoterIdAndDrivingLicenseDropDown" duration-ms="0" started-at="2022-07-18T12:33:42Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.verifyBritishClientNidPassportAndPanCard" finished-at="2022-07-18T12:33:42Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_CatSignOffApplicable_01.verifyNidVoterIdAndDrivingLicenseDropDown()[pri:15, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_CatSignOffApplicable_01.verifyNidVoterIdAndDrivingLicenseDropDown()[pri:15, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyNidVoterIdAndDrivingLicenseDropDown -->
        <test-method status="SKIP" signature="verifyGapReasonApplicableNo()[pri:16, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="verifyGapReasonApplicableNo" duration-ms="0" started-at="2022-07-18T12:33:42Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.verifyNidVoterIdAndDrivingLicenseDropDown" finished-at="2022-07-18T12:33:42Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_CatSignOffApplicable_01.verifyGapReasonApplicableNo()[pri:16, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_CatSignOffApplicable_01.verifyGapReasonApplicableNo()[pri:16, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyGapReasonApplicableNo -->
        <test-method status="SKIP" signature="verifyGapReasonApplicableFormPresent()[pri:17, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="verifyGapReasonApplicableFormPresent" duration-ms="0" started-at="2022-07-18T12:33:42Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.verifyGapReasonApplicableNo" finished-at="2022-07-18T12:33:42Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_CatSignOffApplicable_01.verifyGapReasonApplicableFormPresent()[pri:17, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_CatSignOffApplicable_01.verifyGapReasonApplicableFormPresent()[pri:17, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyGapReasonApplicableFormPresent -->
        <test-method status="SKIP" signature="verifyGapReasonApplicableFormNotPresent()[pri:17, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="verifyGapReasonApplicableFormNotPresent" duration-ms="0" started-at="2022-07-18T12:33:42Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.verifyGapReasonApplicableNo" finished-at="2022-07-18T12:33:42Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_CatSignOffApplicable_01.verifyGapReasonApplicableFormNotPresent()[pri:17, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_CatSignOffApplicable_01.verifyGapReasonApplicableFormNotPresent()[pri:17, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyGapReasonApplicableFormNotPresent -->
        <test-method status="SKIP" signature="verifyEnpAdminLoginAccount()[pri:18, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="verifyEnpAdminLoginAccount" duration-ms="0" started-at="2022-07-18T12:33:42Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.verifyGapReasonApplicableFormPresent" finished-at="2022-07-18T12:33:42Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_CatSignOffApplicable_01.verifyEnpAdminLoginAccount()[pri:18, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_CatSignOffApplicable_01.verifyEnpAdminLoginAccount()[pri:18, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyEnpAdminLoginAccount -->
        <test-method status="SKIP" signature="verifyGapReasonApplicableYes()[pri:18, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447]" name="verifyGapReasonApplicableYes" duration-ms="0" started-at="2022-07-18T12:33:42Z" depends-on-methods="com.iBridge.testcases.TC_CatSignOffApplicable_01.verifyNidVoterIdAndDrivingLicenseDropDown" finished-at="2022-07-18T12:33:42Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_CatSignOffApplicable_01.verifyGapReasonApplicableYes()[pri:18, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_CatSignOffApplicable_01.verifyGapReasonApplicableYes()[pri:18, instance:com.iBridge.testcases.TC_CatSignOffApplicable_01@3fb6a447] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyGapReasonApplicableYes -->
      </class> <!-- com.iBridge.testcases.TC_CatSignOffApplicable_01 -->
    </test> <!-- iBridge18 -->
  </suite> <!-- iBridge -->
</testng-results>
